ConsoleProject/src/view/Frontend.cs

```diff
@@ -125,13 +126,34 @@ public class Frontend {
             };
             return listBox;
         }
+        
+        private void ParseNewFile(string[] fileList) {
```

Comment:
This method should not be in the FrontEnd layer. A better place would be in the TextureConverter class.

Reply:
Moving it to TextureConverter makes it more difficult to interact with Frontend and populate the "images" list only if the file is valid though. I'll see what I can do...

Reply:
So... I moved it here:
![image](https://user-images.githubusercontent.com/23122727/93709020-d2b16880-fb32-11ea-8da0-42e68d23be5b.png)

However, since I cannot interact with the Frontend, the conditions have to be based on the return value of the method, so it means that I have double (triple?) the code in Frontend to achieve the same result as before:
![image](https://user-images.githubusercontent.com/23122727/93709036-faa0cc00-fb32-11ea-8ebe-7f27e97f8b3d.png)